src/goto-checker/goto_checker.h

```diff
@@ -59,6 +59,11 @@ class goto_checkert
   /// This builds and returns the counterexample
   virtual goto_tracet build_error_trace() const = 0;
 
+  /// This outputs an error witness
```

Comment:
Drop Doxy comments that just repeat the function name? Or, provide a link to define witness?